Framework/NexusGeometry/src/NexusGeometrySave.cpp

```diff
@@ -34,6 +34,26 @@ namespace Mantid {
 namespace NexusGeometry {
 namespace NexusGeometrySave {
 
+inline H5::Group safeCreateGroup(const H5::Group &parentGroup,
```

Comment:
Good attempt at naming, but I wouldn't call something `safe...` if it did throw. Maybe `try...` instead?
* Why is this inline?
* Can you doxygen comments in place explaining that this throws.

Reply:
im using inline since i call it multiple times, and i figured it was sufficiently small a function? added doxygen